addon/-private/models/localized.js

```diff
@@ -1,23 +1,24 @@
-import Model from "@ember-data/model";
-import { deprecate } from '@ember/debug';
+import { deprecate } from "@ember/debug";
 import { inject as service } from "@ember/service";
+import Model from "@ember-data/model";
 import { tracked } from "@glimmer/tracking";
 
 export default class LocalizedModel extends Model {
   @service intl;
   @tracked localizedFieldLocale;
 
   getUnlocalizedField(field) {
-    deprecate('Usage of `getUnlocalizedField` is deprecated. Access object directly.',
+    deprecate(
+      "Usage of `getUnlocalizedField` is deprecated. Access object directly.",
       false,
       {
-        id: 'ember-localized-model.public-unlocalized',
-        until: '2.0.0',
-        url: 'https://github.com/projectcaluma/ember-localized-model/pull/101',
-        for: 'ember-localized-model',
+        id: "ember-localized-model.public-unlocalized",
+        until: "2.0.0",
+        url: "https://github.com/projectcaluma/ember-localized-model/pull/101",
+        for: "ember-localized-model",
         since: {
-          enabled: '1.2.0',
-        }
+          enabled: "1.2.0",
+        },
```

Comment:
Since this change is breaking anyways, would it make sense to drop all deprecations in the code as well? Of course, this shouldn't affect this PR - but we should open another one dropping all deprecations.

Reply:
Already did so: https://github.com/projectcaluma/ember-localized-model/pull/151

Reply:
Will review in #151

Reply:
@anehx I'll mark it as resolved for now so we can merge this pr. If you still have changes i'll open a new pr to address them.